uber/custom_tags.py

```diff
@@ -445,10 +445,12 @@ def render(self, context):
 class single_day_prices(template.Node):
     def render(self, context):
         prices = ''
+        skiprest = ''
```

Comment:
I think this was a bugfix of some sort, but it looks a little odd. Need to re-evaluate it for inclusion in master.